docs/2.9.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -194,6 +205,27 @@ The ping command creates two contracts between the admin parties, then exercises
 Version Specific Notes
 ~~~~~~~~~~~~~~~~~~~~~~
 
+.. _upgrade_to_2.9:
+
+Upgrade to Release 2.9
+^^^^^^^^^^^^^^^^^^^^^^
+
+TODO
+
+Other
+"""""
+Version 2.9 adds a new repair command :ref:`participant.repair.purge_deactivated_domain <repair.purge_deactivated_domain>`
+to delete data of a defunct domain.
+
+We recommend to use this command to remove remaining but useless data of previous
+sync domains that have been migrated using the :ref:`participant.repair.migrate_domain <repair.migrate_domain>`
+command.
+
+Note that the ``migrate_domain`` command in 2.9 now automatically removes such data,
+but only for the sync domain on which it has been invoked on.
```

Comment:
The details of the changes are in the release notes. In that page, we only list changes that require changes from the end users (adapt config, change a console command, ...) Since the purge_deactivated_domain is not needed for the migration (invoked transparently), I'd remove this new section for now.

Reply:
> Since the purge_deactivated_domain is not needed for the migration (invoked transparently), I'd remove this new section for now.

I added this section specifically to point out that customers should run the command for domains which they have migrated before as I think their data still lingers in the data store. When doing a hard domain migration only the old (source) domain gets cleaned up, but other inactive (deactivated) domains remain as is.

We added this command specifically for that purpose.